### ISSUE TITLE
How to run this code?

### ISSUE THREAD

**Issue by mwy691222563** [szad670401/Fast-MTCNN#8]
usr/local/include/opencv2/core/cvstd.hpp:556:5: error: ‘cv::String::String(int)’ is private
     String(int); // disabled and invalid. Catch invalid usages like, commandLineParser.has(0) problem
     ^
In file included from /usr/include/c++/5/bits/stl_tempbuf.h:60:0,
                 from /usr/include/c++/5/bits/stl_algo.h:62,
                 from /usr/include/c++/5/algorithm:62,
                 from /usr/local/include/opencv2/core/base.hpp:55,
                 from /usr/local/include/opencv2/core.hpp:54,
                 from /usr/local/include/opencv2/opencv.hpp:52,
                 from mtcnn_opencv.cpp:4:
/usr/include/c++/5/bits/stl_construct.h:83:7: error: within this context
       ::new(static_cast<void*>(__p)) _T1(__value);

**Comment by graycrown** [comment]
I am in the same condition, have you solved it?

**Comment by graycrown** [comment]
If you run this code on ubuntu , try   "g++ -o mtcnn_opencv mtcnn_opencv.cpp -std=c++11", it seems like that the error is caused by C++11 new feature.

**Comment by jhgfkdj** [comment]
@graycrown @mwy691222563  Please help me to run this code. I use cmake and make way to generate it, but errors ocured:

/mtcnn_opencv.cpp.o：在函数‘MTCNN::MTCNN(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)’中：
mtcnn_opencv.cpp:(.text+0xae)：对‘cv::dnn::experimental_dnn_v4::Net::Net()’未定义的引用
mtcnn_opencv.cpp:(.text+0xc1)：对‘cv::dnn::experimental_dnn_v4::Net::Net()’未定义的引用
mtcnn_opencv.cpp:(.text+0xd4)：对‘cv::dnn::experimental_dnn_v4::Net::Net()’未定义的引用
mtcnn_opencv.cpp:(.text+0x174)：对‘cv::dnn::experimental_dnn_v4::readNetFromCaffe(cv::String const&, cv::String const&)’未定义的引用
mtcnn_opencv.cpp:(.text+0x19c)：对‘cv::dnn::experimental_dnn_v4::Net::~Net()’未定义的引用
....

I dont know where the wrong is.
If it's the path problem, I did setted and included the opencv path.

**Comment by mwy691222563** [comment]
it seems some libs should be linked.

**Comment by jhgfkdj** [comment]
@mwy691222563 yeah I found some opencv libs should be linked in CMakeLists file. thx~~

**Comment by scutzhe** [comment]
It's the same quesion @graycrown  how do you solve

**Comment by Linson93** [comment]
@jhgfkdj  excuse me ,how to do it ?
